src/openms/source/CHEMISTRY/ResidueDB.cpp

```diff
@@ -191,7 +194,7 @@ namespace OpenMS
         }
       }
     }
-    buildResidueName_(r);
+    buildResidueNames_();
```

Comment:
I think it might be possible to use the buildResidueName_(r) which should be much faster.
Maybe just on the non-modified ones? Need to check.